src/openloco/openloco.cpp

```diff
@@ -561,6 +563,74 @@ namespace openloco
         }
     }
 
+    static void sub_496A84(int32_t edx)
+    {
+        // This is responsible for updating the snow line
+        registers regs;
+        regs.edx = edx;
+        call(0x00496A84, regs);
+    }
+
+    // 0x004968C7
+    static void date_tick()
+    {
+        if ((addr<0x00525E28, uint32_t>() & 1) && !is_editor_mode())
+        {
+            if (update_day_counter())
+            {
+                call(0x0048B244);
+                call(0x004B94CF);
+                call(0x00453487);
+                call(0x004284DB);
+                call(0x004969DA);
+                call(0x00439BA5);
+
+                auto yesterday = calc_date(current_day() - 1);
+                auto today = calc_date(current_day());
+                set_date(today);
+                sub_496A84(today.day_of_olympiad);
+                if (today.month != yesterday.month)
+                {
+                    // End of every month
+                    addr<0x0050A004, uint16_t>() += 2;
+                    addr<0x00526243, uint16_t>()++;
+                    call(0x0049748C);
+                    call(0x0045383B);
+                    call(0x0043037B);
+                    call(0x0042F213);
+                    call(0x004C3C54);
+
+                    if (today.year <= 2029)
+                    {
+                        call(0x0046E239);
+                    }
+
+                    // clang-format off
```

Comment:
I'd rather we improve the format if this is treated wrongly.

Reply:
Is it possible to make clang-format respect the line breaks?